public/profile.html

```
@@ -0,0 +1,680 @@
+              <div id="avatar-circle" class="avatar-ring w-24 h-24 rounded-full overflow-hidden bg-gradient-to-br from-teal-400 to-cyan-500 flex items-center justify-center text-white text-3xl font-bold select-none">
+                <span id="avatar-initials">?</span>
+                <img id="avatar-img" class="hidden w-full h-full object-cover" alt="avatar" />
+              </div>
+              <input id="avatar-file-input" type="file" accept="image/jpeg,image/png,image/webp" class="hidden" onchange="handleAvatarFile(this)" />
+            </div>
+
+            <!-- Avatar actions -->
+            <div class="flex-1 space-y-3">
+              <p class="text-sm text-gray-500 dark:text-gray-400">Upload a photo to use as your profile picture.</p>
+              <div class="flex flex-wrap gap-2">
+                <button onclick="document.getElementById('avatar-file-input').click()"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)